### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by brianmcmichael**
Updates to readme and documentation for current state.

### CODE REVIEW

**Comment by brianmcmichael** [comment]
This is a documentation update, failing checks are addressed in #72

**Review by hexonaut — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -44,9 +44,9 @@ The spell itself is deployed as follows:
 
 ```js
 new DssExec(
-    "A test dss exec spell",                // Description
-    now + 30 days,                          // Expiration
-    address(new SpellAction(execlib, true)) // true = officeHours on
+    "A test dss exec spell",      // Description
+    now + 30 days,                // Expiration
+    address(new SpellAction())    // true = officeHours on
```
Comment (gbalabasquer) [inline README.md:49]: Remove `true = officeHours on`

**Review by gbalabasquer — APPROVED** [review]
Just a minor change requested, the rest looks good

**Pull request merged**